java/benchmarks/src/main/java/glide/benchmarks/utils/Benchmarking.java

```diff
@@ -24,6 +29,8 @@ public class Benchmarking {
     static final int SIZE_SET_KEYSPACE = 3000000;
     public static final double NANO_TO_SECONDS = 1e9;
 
+    static final int NUM_OF_THREADS_TO_EXECUTE = 12;
```

Comment:
Do you need to bind this to the num of CPUs? What if it is executed on a VN with 1 CPU or with 128 CPUs?

Reply:
This is not used.  Removing.